.github/dependabot.template.yml

```diff
@@ -25,16 +25,18 @@ updates:
     directory: /domains/*/*.sln
     schedule:
       interval: daily
-    groups:
-      "Microsoft.EntityFrameworkCore":
-        patterns:
-          - "Microsoft.EntityFrameworkCore.*"
-
+    ignore:
+      - dependency-type: "all"
+        update-types: 
+        - "minor"
+        - "patch"
+    
```

Comment:
Hvad er det vi håber at opnå med denne opdatering.

Reply:
At vi ikke får åbnet pull requests på opdatering af minor/path versioner af dependencies med mindre der er sikkerhedsfejl som er blevet rettet.

Reply:
Hvorfor ikke blot filtrere i Pull request listen? så undgår vi at se dem.

Reply:
Så vil vi i det mindste ikke se dem. Men har vi overhovedet brug for at de bliver oprettet som pull requests?

Det vil være fint med overblikket over opdateringer... men det virker som overkill af oprette en PR på hver og en af dem?